src/ci_rdm/ci_rdms.h

```diff
@@ -244,6 +258,12 @@ class CI_RDMS {
 
     // Generate three-particle map
     void get_three_map();
+
+    // Fill 3rdm
+    void fill_3rdm( std::vector<double>& tprdm, double value, int p, int q, int r, int s, int t, int u , bool half = false);
+    void make_ab(std::vector<double>& tprdm_aab, std::vector<double>& tprdm_abb );
+    void make_ab(SortedStringList_UI64 a_sorted_string_list_,const  std::vector<UI64Determinant::bit_t>& sorted_astr,const std::vector<UI64Determinant>& sorted_a_dets, 
+                std::vector<double>& tprdm_ab, std::vector<double>& tprdm_aab,std::vector<double>& tprdm_abb);
```

Comment:
Comment these functions a bit more argument. It would be nice to leave a note on storage format of RDMs somewhere in this code.